contrib/core/actions/send_mail/send_mail

```diff
@@ -25,7 +25,13 @@ SEND_EMPTY_BODY=$1
 shift
 CONTENT_TYPE=$1
 shift
-BODY="$@"
+BODY="$1"
+shift
+IFS=',' read -ra ATTACHMENTS <<< "$@"
+
+if [[ $ATTACHMENTS == "None" ]]; then
```

Comment:
Without this guard, it tries to find an attachment named "None". This can be seen on the `stderr` output.
(Should "None" be passed to a `required: false` positional parameter? What is normally passed?)

```
    stderr: 'Warning: attachment None not found, skipping'
```